site/docs/pages/index.mdx

```diff
@@ -121,13 +196,20 @@ bun add @coinbase/onchainkit
       </App>
       </div>
     </div>
+
   </aside>
-  <aside className="md:px-2 pt-10">
-    <p className="text-lg md:text-xl py-4">
+  <main id="wallet" className="pt-[56px] flex flex-col gap-4 md:flex-row md:text-left text-center">
+    <h3 className="basis-1/2 py-[15px] text-4xl md:text-4xl">Wallet</h3>
+  </main>
+  <aside className="pb-6">
+    <p className="text-base md:text-base pb-[24px] md:text-left text-center">
       Create or connect your wallet with [Connect Wallet](/wallet/wallet), powered by [Smart Wallet](https://www.smartwallet.dev/why).
     </p>
-    <div className="flex grow flex-col md:flex-row">
-      <div>
+    <a href="/wallet/wallet" className="mx-auto md:m-0  flex justify-center items-center border border-indigo-400 text-indigo-400 font-bold leading-6 px-4 py-3 rounded-xl border-solid w-[180px]">
```

Comment:
@abcrane123 As follow up we should triple check the SEO miss, we might have, by running our page for web and mobile on https://pagespeed.web.dev/